docs/advanced_applications/ICL.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 2
+---
+
+# 🟡 Advanced ICL
+
+ICL is an exciting new paradigm in NLP where large language models LLMS make predictions based on contexts augmented with just a few training examples. LLM are able to extract patterns from the examples provided in the context and use them to perform many complex NLP tasks. We'll be discussing recent optimizations of ICL covering methods for fine-tuning LLM for ICL, how to pick good training examples for ICL and how to design great prompts.
```

Comment:
Say in context learning (ICL) and add vocabulary hint (see https://learnprompting.org/contribute#prompt-engineering-techniques).